dotnet/src/Agents/OpenAI/Internal/AssistantToolResourcesFactory.cs

```diff
@@ -26,18 +26,20 @@ internal static class AssistantToolResourcesFactory
 
         if (hasVectorStore || hasCodeInterpreterFiles)
         {
-            var fileSearch = hasVectorStore
-                ? new FileSearchToolResources
-                {
-                    VectorStoreIds = { vectorStoreId! }
-                }
-                : null;
-
-            var codeInterpreter = hasCodeInterpreterFiles
-                ? new CodeInterpreterToolResources()
-                : null;
+            FileSearchToolResources? fileSearch =
+                hasVectorStore ?
+                    new()
+                    {
+                        VectorStoreIds = { vectorStoreId! }
+                    } :
+                    null;
 
+            CodeInterpreterToolResources? codeInterpreter =
+                hasCodeInterpreterFiles ?
+                    new() :
+                    null;
             codeInterpreter?.FileIds.AddRange(codeInterpreterFileIds!);
+
```

Comment:
Opportunistic update of `var` convention from external update.  Unable to apply suggestion at time since it spanned deleted lines.  (no functional change)